homeassistant/components/deconz/climate.py

```diff
@@ -221,6 +225,21 @@ async def async_set_hvac_mode(self, hvac_mode: str) -> None:
 
         await self._device.set_config(**data)
 
+    @property
+    def hvac_action(self) -> str:
```

Comment:
The parent class defines return type as str | None

Looking through the different properties you base the action on most can be None. In the case of a Tuya thermostat I have in my tests in pydeconz both mode and state_on are None and preset "auto".

I guess it could be a bad example as "lastupdated": "none".

What I mean is, could there be a situation here where the proper thing would be to return None rather than a string?